framework/doc/content/source/vectorpostprocessors/ElementMaterialPropertySampler.md

```diff
@@ -1,27 +1,27 @@
-# MaterialVectorPostprocessor
+# ElementMaterialPropertySampler
```

Comment:
Did somebody complain about this name? There are definitely pros to adding more clarity to a name, but there is also a con of making users update their inputs. A lot of renaming can lead to update fatigue

Reply:
just me. I like the names to follow convention
someone wanted the equivalent of a BoundaryMaterialPropertySampler the other day and it would be orderly for them to see the Element...Sampler to make the Boundary one

Reply:
"Are you suffering from update fatigue? Ask your doctor about..."

I agree that there's such a thing as too much renaming, but I'd argue this one does bring enough clarity to be justified. We just need to make sure to get it right this time. I see we have `LineMaterialRealSampler`. Maybe just drop "property" from the name?

Reply:
@YaqiWang who does the brunt of the renaming for Griffin?

Reply:
So ElementMaterialRealSampler ?

Reply:
ah no

Reply:
it does other properties than Real

as you want!

Reply:
I ll change to ElementMaterialSampler at 5pm MT unless advised otherwise?

Reply:
I'm on board for whatever the people decide